### PR TITLE
CLOUDP-181546: slow query logs

### PR DESCRIPTION
**Pull request by wtrocki**
## Proposed changes

Fixing structure changes that were missing in the go template.

### CODE REVIEW

**Comment by wtrocki** [comment]
Slow Query logs seem to take time to appear as items. Looking for the way to improve integration test as it bases only on the empty response right now.

**Comment by wtrocki** [comment]
After investigating problem I see that it will be quite challenging for us to simulate any responses in slowQueryLogs due to the fact that it takes some time to make them and collect them thru API.

File: internal/cli/atlas/performanceadvisor/slowquerylogs/list.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 	atlasv2 "go.mongodb.org/atlas-sdk/admin"
 )
 
-const listTemplate = `NAMESPACE	LINE{{range .SlowQuery}}
+const listTemplate = `NAMESPACE	LINE{{range .SlowQueries}}
```
Comment (andreaangiolillo) [inline internal/cli/atlas/performanceadvisor/slowquerylogs/list.go:31]: Could you add a test to check that the template is correct? You can use this PR as an example https://github.com/mongodb/mongodb-atlas-cli/pull/1471

Comment (wtrocki) [reply internal/cli/atlas/performanceadvisor/slowquerylogs/list.go:31]: Awesome suggestion! We already have unit test for each command but that does use raw json.
I'm investigating if we can do this for all commands with little effort track those issues in the future.

**Review by matt-condon — DISMISSED** [review]

File: internal/cli/atlas/performanceadvisor/slowquerylogs/list.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/atlas/performanceadvisor/slowquerylogs/list.go:31]: there is a ticket in the backlog to add this check to all the unit tests

**Review by swattyT — APPROVED** [review]
LGTM!

**Review by swattyT — APPROVED** [review]
LGTM!

**Review by gssbzn — APPROVED** [review]

**Pull request merged**